packages/drawer/src/styled/transitions.tsx

```diff
@@ -59,7 +60,7 @@ HTMLElement, ITransitionProps & RefAttributes<HTMLElement>
       transition: `opacity ${transitionDurationMs}ms ${transitionTimingFunction}`,
       opacity: 0,
       position: 'fixed',
-      zIndex: 0,
+      zIndex: props.zIndex ?? getElementStyles(props.theme, 'drawerWrapper', props.appearance, props.baseAppearance).zIndex ?? 'auto',
```

Comment:
zIndex property can be integer or 'auto', but interface is described as only integer.

Reply:
zIndex css property can be auto, but props zIndex for component can't be auto